sdk/tests/utilities/portfolio_loader.py

```diff
@@ -0,0 +1,73 @@
+import lusid
+import lusid.models as models
+from utilities import InstrumentLoader
+import uuid
+
+
+class PortfolioLoader:
+
+    def __init__(self, transaction_portfolios_api: lusid.TransactionPortfoliosApi, instruments_api: lusid.InstrumentsApi):
+        self.transaction_portfolios_api = transaction_portfolios_api
+        self.instruments_api = instruments_api
+
+    def load_instruments(self, instruments_api) -> list:
+
+        instrument_loader = InstrumentLoader(instruments_api)
+        return instrument_loader.load_instruments()
+
+    def build_transaction_request(self, instrument_id, id_type, units, price, currency, trade_date, transaction_type):
```

Comment:
is this not covered by https://github.com/finbourne/lusid-sdk-python-preview/blob/master/sdk/tests/utilities/test_data_utilities.py

Reply:
Agreed, needed a slight refactor though, as needed to use other id_type than a luid.